### DISCUSSION TITLE
Convert old db files to .sql?

### DISCUSSION THREAD

**Discussion by aaronrobb** [leokhoa/laragon#453] [Q&A]
I have an old laragon install on an old machine.
I'm looking to move the old database to mysql but all I'm finding are the .frm and .idb files.
Is there an easy way to convert this to a .sql file?

**Comment by aaronrobb** [comment]
I just re-setup Laragon and got into the database that way.

**Reply by osamalmezouari** [reply]
yes it's work thanks
